### ISSUE TITLE
Display issues

### ISSUE THREAD

**Issue by enrixp** [smartperson/spartan-edge-accelerator-graphical-system#10]
1. I have tested it on a Samsung tv HDMI is not recognising and signal is not stable .
However on a hdmi capture card on my laptop the signal is fine .
2. Is there a way to include the code on esp32 for default bitstream upload and web server as well so we don't have to use the jtag .

**Comment by smartperson** [comment]
I created #11 to track the bitstream question.

**Comment by smartperson** [comment]
I've seen some irregularity on Samsung TVs. One thing I found causes problems is if the Samsung TV is in "Game Mode" which probably depends on extremely reliable timing signals. We're pushing the frequency range of this board with 720p/1080p signals, as the TMDS clock is running at ~10x the system's internal pixel clock.

Right now the video output from this hardware actually uses DVI signals. We're kind of cheating because HDMI includes a DVI compatibility mode. Unsurprisingly, this mode can cause issues.

This might be the issue I use to track eventually incorporating a proper HDMI solution. It doesn't sit high on my backlog, but it's 100% a worthwhile project. https://hackaday.com/2020/06/02/an-open-source-hdmi-implementation-for-fpgas/

**Comment by enrixp** [comment]
Yes that's what I was thinking as well . I try to use the proper hdmi code as well which requires a signal from the camera port rpi camera and it works fine .
Regarding the esp32 yesterday I uploaded the bitstream on the piffs and it works fine without men card and jtag
